src/components/PlotWrapper.tsx

```diff
@@ -37,10 +40,24 @@ export default function PlotWrapper(inputProps: PlotWrapperProps): ReactElement
     }
   }, [props.dataset]);
 
-  // Update time in plot
+  // Update time and hovered value in plot
   useMemo(() => {
-    plot?.setTime(props.frame);
-  }, [props.frame]);
+    const config: TrackPlotLayoutConfig = {
+      time: props.frame,
+    };
+    if (hoveredObjectId) {
+      const featureData = props.dataset?.getFeatureData(props.featureKey);
+      const hoveredObjectTime = props.dataset?.getTime(hoveredObjectId);
+      if (featureData && hoveredObjectTime) {
+        const featureValue = featureData.data[hoveredObjectId];
+        config.hover = {
+          x: hoveredObjectTime,
+          y: featureValue,
+        };
+      }
+    }
```

Comment:
Entirely optional thought: consider reordering this `if` and the definition of `config`, adding a `let hover: { x: number, y: number } | undefined = undefined;` (type extra optional), and using `hover` in the definition of `config`:

```typescript
const config: TrackPlotLayoutConfig = {
  time: props.frame,
  hover,
};
```

Makes the structure of the value you're creating a bit more obvious, without referencing the definition of `TrackPlotLayoutConfig`